### ISSUE TITLE
[BUG] [VCC-WEB] RepoList is not loaded.

### ISSUE THREAD

**Issue by ni1chigo2115** [vrchat-community/creator-companion#119]
### Describe the  bug
RepoList is not loaded.
**Furthermore, the situation has worsened since the previous version: it is no longer possible to retrieve information on packages related to UserRepo, and package updates cannot be performed from within VCC.**
https://github.com/vrchat-community/creator-companion/issues/112
(The situation does not change even after some time has passed since loading was displayed.)

### To Reproduce

- Close VCC (to avoid another issue https://github.com/vrchat-community/creator-companion/issues/111)
-**add two or more userRepos** in settings.json in some way (you may edit settings.json or use vpm cli)
-Open VCC
-Select project
-Repolist does not change in the Loading state.

### Expected behavior
load UserRepoList so that User's VPM assets can be updated from VCC.

### Screenshots
![vrrhtml5](https://user-images.githubusercontent.com/23652237/224199043-2594a867-042e-4fef-8a40-4f64740b60c1.png)

### Desktop (please complete the following information):
OS: Windows 10 Pro
Version 22H2  Build 19045.2604

### Additional context
It seems that if you have added even one User Repo, the RepoList will not load, and to be honest, it seems to have gotten even worse since the last version.
I honestly don't think it will show up even if I wait an hour.

**In particular, unlike the previous bug that froze and took a long time to load, this time the bug is even worse, as it will not load in the first place and will not allow User Repo assets to be updated.**
**Currently, managing UserRepo by registering URLs in VCC requires knowledge of CMD, which is a difficult task, and there are not many users. However, considering the future management of avatar projects, it is absolutely necessary to develop this system.**

**Comment by anatawa12** [comment]
I found that this will happen if we're selecting multiple user repos. If we're selecting only one user repo, this problem doesn't happen.

**Comment by orels1** [comment]
Hello! The user repos are not fully supported in the user interface just yet. This is our main goal now that the new VCC 2.0 is fully public, so right this moment, this functionality is expected to be unstable.

**However**, we understand that regressions are not a good experience for anyone and thus would like to address this with a fix.

The limitation of the VCC 2.0 is to only load the packages for the **first** checked custom repo. However it should not prevent you from loading the package manager, here's an example repo URL that should work 

`https://vrchat-community.github.io/template-package/index.json`

It shows up in the list as `VRChat Example Package` and you can add/remove it from the list

![image](https://user-images.githubusercontent.com/3798928/224309900-50bdc816-811d-4165-bd02-fabcd613d887.png)

If you can give us repo links that cause your VCC to be stuck in an infinite loading - that would help us address the issue, as so far I am unable to reproduce the bug

**Comment by ni1chigo2115** [comment]
>If you can give us repo links that cause your VCC to be stuck in an infinite loading - that would help us address the issue, as so far I am unable to reproduce the bug
https://vpm.nadena.dev/vpm.json
https://vpm.anatawa12.com/vpm.json
https://lilxyzw.github.io/vpm-repos/vpm.json

These are the three points for now.
In particular, anatawa's Repo implements multiple tool packages.

**Comment by ni1chigo2115** [comment]
Also, I have 'multiple UserRepo's checked' before updating. (This may be particularly suspicious since I basically check all of them once I put them in.)

**Comment by orels1** [comment]
did you add them all in settings or via CLI? I just added them all with CLI, and I can see the 
```
Could not connect to remote repo. Loading from disk instead. Exception: The process cannot access the file 'C:\Users\orel-\AppData\Local\VRChatCreatorCompanion\Repos\39bc8d4a-c08d-44aa-b321-cc3b6f2f2d2d.json' because it is being used by another process.
```

Error in the log sometimes.

Do you have any similar errors in your VCC log when loading packages?

I think I can see the issues where the VCC attempts to perform so many filesystem updates to the JSON file while resolving the repo - it gets stuck. Force-reloading the VCC using Ctrl+R shortcut does often help, but that's not a good experience, and it should just abort the loading when a repo fails to resolve instead of getting "stuck". 

Thanks for those links! They helped me find the potential crash source.

> I feel that we should pay attention to the fact that when we move from VCC1 to VCC2, we inherit and update the settings as they are.

Unfortunately, the way data was stored fundamentally changed between VCC 1 and 2, and considering that the repo functionality was never officially launched in full, those settings were not transferred over.

**Comment by ni1chigo2115** [comment]
Only CLI(VCC1.0x)

**Comment by orels1** [comment]
Seems like we have a fix.
![image](https://user-images.githubusercontent.com/3798928/224366799-97a657a2-3ebb-40bd-82ac-83743247588c.png)

We'll see if we can ship it out later today

**Comment by orels1** [comment]
This should be fixed in the v2.0.1 that just went live! You should be able to launch the VCC, go to settings and click "Check For Updates" to get the update immediately.

If you still have issues, please open a new issue with more details.
Thanks for the report!

**Comment by ni1chigo2115** [comment]
We have confirmed the correction. Thank you!
This time it was the first thing I saw and it was critical, so I had it fixed as a top priority, but if there are any more problems with VCC 2.0, I would like to contact you again.
![fixwork](https://user-images.githubusercontent.com/23652237/224470234-211e4670-0e73-49a9-91e8-beec99c6f7d0.jpg)
